### DISCUSSION TITLE
Nextcloud Container Not starting on Unraid

### DISCUSSION THREAD

**Discussion by 00o-sh** [nextcloud/all-in-one#1415] [Questions] [answered]
this is what I see in the logs
![image](https://user-images.githubusercontent.com/85048617/202238613-9f9cc2ba-5f08-4aa3-b4cd-afd374a2677c.png)
can someone please advise

**Comment by szaimen** [comment]
Is this the first time you are trying to use AIO?

**Reply by 00o-sh** [reply]
Yes 
I may be missing something this is going thru cloudflare tunnel

**Comment by szaimen** [comment, marked as answer]
Did you see this? 
![image](https://user-images.githubusercontent.com/42591237/202428223-1d10b378-0d74-468b-a235-ab50ce81882d.png)
https://github.com/nextcloud/all-in-one#how-to-change-the-default-location-of-nextclouds-datadir
